internal/view/assets/js/page/home.js

```diff
@@ -702,15 +702,36 @@ export default {
 						this.editMode = false;
 						this.dialog.loading = false;
 						this.dialog.visible = false;
-
-						json.forEach(book => {
-							var item = items.find(el => el.id === book.id);
-							this.bookmarks.splice(item.index, 1, book);
-						});
-					}).catch(err => {
-						this.selection = [];
-						this.editMode = false;
-						this.dialog.loading = false;
+                        
```

Comment:
I think that to make this simpler for now (since we don't have a nice way of presenting this to the user) I would just expose if everything went right or not.

The error message could look something like `Not all bookmarks could have their contents updated, but no files were overwritten.`

In order to have some debug information, we could expose the IDs in a single `console.error()` statement along with a server log, maybe?